CesiumGltfReader/src/decodeDataUrls.cpp

```diff
@@ -126,10 +126,13 @@ void decodeDataUrls(
 
     ImageReaderResult imageResult =
         reader.readImage(decoded.value().data, options.ktx2TranscodeTargets);
-    if (imageResult.image) {
-      image.cesium = std::move(imageResult.image.value());
+
+    if (!imageResult.image) {
+      continue;
     }
 
+    image.cesium = std::move(imageResult.image.value());
+
     if (options.clearDecodedDataUrls) {
       image.uri.reset();
     }
```

Comment:
This change ensures that `image.uri` is only reset if the decode succeeds. Otherwise if decoding fails it'll hit the same crash.